### DISCUSSION TITLE
Which Options to keep ON/OFF while whitelisting site to get not log out of it.

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [Cookie-AutoDelete/Cookie-AutoDelete#1485] [Q&A / Support]
I read the faq , several times but i can't figure it out.
![image](https://user-images.githubusercontent.com/75798544/214956017-5fdd8fdf-e5dc-49a6-91bc-156dc94127b0.png)
Which one should be ON/OFF , so that i don't get logged out of websites and deleting extra cookies ?

**Comment by kennethtran93** [comment]
Unfortunately there's no one-size-fits-all answer here - it depends on each website in question.  Historically it was primarily cookies, but in the current era there are various storage areas that can be utilized to store login credentials, and some may use more than one areas as well (redundancy/backups/security).

Plugin Data is technically deprecated (Java), so that can be left unchecked.
